src/index.html

```diff
@@ -1,13 +1,428 @@
+                    class="testimonial__photo"
+                    src="images/people/3.jpg"
+                    alt="Leslie Mckinney"
+                  >
+                </div>
+                <h3 class="testimonial__name">
+                  <a href="#" class="link">Max Henry</a>
+                </h3>
+
+                <div class="testimonial__role">
+                  Chief Design Officer
+                </div>
+                <p class="testimonial__about">
+                  When my lovely valley of steam rises and half-day
+                  sun is above an impermeable
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="page__section contact-us"
+               id="contact-us">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-2">
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Phone
+                </div>
+                <a
+                  href="tel:+1 234 5555-55-55"
+                  class="contact-us__info-link"
+                >
+                  +1 234 5555-55-55
+                </a>
+              </div>
+
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Email
+                </div>
+                <a
+                  href="mailto:hello@miami.com"
+                  class="contact-us__info-link"
+                >
+                  hello@miami.com
+                </a>
+              </div>
+            </div>
+            <div class="grid__item grid__item--4-5">
+              <div class="contact-us__info contact-us__info--last">
+                <div class="contact-us__info-title">
+                  Address
+                </div>
+                <a
+                  rel = "noopener"
+                  href="https://goo.gl/maps/A6swfZ4NjbvztNoFA"
+                  class="contact-us__info-link"
+                  target="_blank"
+                >
+                  400 first ave. n.
+                  <br>
+                  suite 700
+                  <br>
+                  Minneapolis, MN
+                  <br>
+                  55401
+                </a>
+              </div>
+            </div>
+            <div class="grid__item grid__item--8-12">
+              <form action="" method="get">
+                <input
+                  class="contact-us__field form-field"
+                  name="name"
+                  type="text"
+                  placeholder="Your name"
+                >
+                <input
+                  class="contact-us__field form-field"
+                  name="email"
+                  type="email"
+                  placeholder="Example@mail.com"
+                >
```

Comment:
It'd be better to make these fields required

Reply:
don't understand